### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by kumaresankandasamy**

### CODE REVIEW

**Comment by kumaresankandasamy** [comment]
update done

File: index.js [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
+promise.then(res=>res.json())
+       .then(d=>updateDOM(d.data))
+       .catch(error=>console.log(error))
+
+var updateDOM=function(data){
+	var username=document.querySelector('div.username')
+	username.innerHTML=data.full_name;
+	/*var userid=document.querySelector('div.userid')
+	userid.innerHTML=data.id; */
+	var tweets=document.querySelector('p#tweets')
```
Comment (manjariParthiban) [inline index.js:11]: did not update right data

File: index.js [hunk 2]
```diff
@@ -1,41 +1,44 @@
...
+	tweets.innerHTML=data.stats.tweets;
+	var following=document.querySelector('p#following')
```
Comment (manjariParthiban) [inline index.js:13]: did not update right data

File: index.js [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
+	following.innerHTML=data.stats.following;
+	var followers=document.querySelector('p#followers')
```
Comment (manjariParthiban) [inline index.js:15]: did not update right data

File: index.js [hunk 4]
```diff
@@ -1,41 +1,44 @@
...
+	followers.innerHTML=data.stats.followers;
+	var sidebarusername=document.querySelectorAll('div.follow-body span.username')
```
Comment (manjariParthiban) [inline index.js:17]: should have used different api. Did not update right data

File: index.js [hunk 5]
```diff
@@ -1,41 +1,44 @@
...
+	var promise2=fetch("https://fsd1.herokuapp.com/users/1/following/tweets");
+	promise2.then(res=>res.json())
+       .then(d=> updateTweet(d.data))
+       .catch(error=>console.log(error))
+var updateTweet=function(data){
+	for(var i=0;i<=data.length;i++){
+	var tweettext=document.querySelector('div.comment-body p')
+		tweettext.textContent=data[i].text;
+		document.querySelector('div span#comment').
+		appendChild(document.createTextNode(data[i].stats.comments));
```
Comment (manjariParthiban) [inline index.js:36]: wrong value due to append child

File: index.js [hunk 6]
```diff
@@ -1,41 +1,44 @@
...
+		document.querySelector('div span#heart').
+		appendChild(document.createTextNode(data[i].stats.likes));
```
Comment (manjariParthiban) [inline index.js:38]: wrong value due to append child

File: index.js [hunk 7]
```diff
@@ -1,41 +1,44 @@
...
+		document.querySelector('div span#refresh').
+		appendChild(document.createTextNode(data[i].stats.retweets));
```
Comment (manjariParthiban) [inline index.js:40]: wrong value due to append child

**Pull request merged**